WordPress/Classes/Models/UserSuggestion+Comparable.swift

```diff
@@ -0,0 +1,7 @@
+extension UserSuggestion: Comparable {
+    public static func < (lus: UserSuggestion, rus: UserSuggestion) -> Bool {
```

Comment:
[Nit]
```suggestion
    public static func < (lhs: UserSuggestion, rhs: UserSuggestion) -> Bool {
```

Reply:
I'm curious, what do `lus` and `ldn` stand for?

Reply:
lus: left user suggestion
rus: right user suggestion
ldn: left display name
rdn: right display name
I thought it would be better than the standard `lhs (left-hand side)` and `rhs (right-hand side)`, but I can change to use lhs and rhs. I can apply more meaningful names to the unwrapped display names, also.